lib/resty/checkups/base.lua

```diff
@@ -294,6 +295,10 @@ function _M.extract_servers_from_upstream(skey, cls)
     end
 
     for _, srv in ipairs(srvs) do
+        if ups_skip_down and srv.down then
+            goto continue
```

Comment:
LuaJIT's `goto` primitive cannot across the bound of new variables.

Reply:
OK, I got it.

Reply:
Thanks for your comment. But I found that [goto-scope](http://lua.2524044.n2.nabble.com/Clarification-of-goto-scope-td7241359.html)

According to the content of the email, I think I can use it like this.

Reply:
Well done.